eoscostarica/proton-affiliate
# Feat/mail

### Send confirmation email

### What does this PR do?

- Resolve #318 

### Steps to test

1. Make an Apply now
2. Approve the request
3. Check the mail is received by the account that made the apply
